21120558/meogroup-backend
# Add new files test2.txt and hehe.txt with text and file mode changes

## Pull Request

### Title
Add new files test2.txt and hehe.txt

### Description
This pull request adds two new files, test2.txt and hehe.txt, to the repository.

### Changes
- Added test2.txt with multiple lines of text.
- Added hehe.txt with file mode 100644 and index e69de29.

### Testing
To test the changes:
1. Clone the repository.
2. Checkout this branch.
3. Verify that test2.txt and hehe.txt are present in the repository.

### Related Issues
None

### Checklist
- [ ] Added test2.txt
- [ ] Added hehe.txt
- [ ] Updated README if necessary
